_layouts/prod-layout.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	{% include head.html %}
+
+	<body>
+		<header>
+		{% include navbar.html %}
+		</header>
+
+		{{ content }}
+
+  	{% include footer.html %}
+	</body>
+    {% include script.html %}
```

Comment:
I think when you load in the JS assets, it needs to scripts, then closing body tag.

It could just be a legacy browser thing, but you may look into it.